### ISSUE TITLE
Replicating CIFAR10 results on your paper

### ISSUE THREAD

**Issue by TakeoHosomi** [zbs881314/Temporal-Coded-Deep-SNN#4]
Thank you for sharing your code.
I'm interested in your work ang trying to replicate your CIFAR10 results.
However I have two issues.

1.  I only got 61% when I run TrainCifar10_S.py without loading weights and run TestCifar10_S.py, though pre-trained weights shows 64% when I run  TestCifar10_S.py. Could you let me know how to generate your pre-trained weights from scratch?

2. In your paper, you wrote "Therefore, we limited the maximum allowed row-normalized Frobenius norm of the gradient of each weight matrix to 10." at page 5. Could you let me know which part of the code implement this feature.

**Comment by TakeoHosomi** [comment]
I have modified some and got 69% on TestCifar_S.py:
1. use lr_decay as TrainCifar_L.py
2. use Cifar10_with_data_augmentation.CFAR10() as dataloader
3. disable whiten option at Cifar10_with_data_augmentation.CFAR10
